.github/workflows/ci-test.yml

```diff
@@ -46,14 +46,14 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v4
         with:
-          distribution: temurin
+          distribution: liberica
```

Comment:
How does a different Java implementation impact the journal recovery output?

Reply:
Moved JDK change to PR - https://github.com/eXist-db/exist/pull/5480